vega_sim/fairground_service.py

```diff
@@ -0,0 +1,67 @@
+from typing import Dict, List, Optional, Set
+
+import time
+import subprocess
+
+from os import path
+
+from vega_sim import vega_bin_path, vega_home_path
+from vega_sim.service import VegaService
+from vega_sim.wallet.base import Wallet
+from vega_sim.wallet.slim_wallet import SlimWallet
+from vega_sim.wallet.vega_wallet import VegaWallet
+
+
+def start_wallet_service():
+
+    wallet_args = [
+        "vega",
+        "wallet",
+        "service",
+        "run",
+        "--network",
+        "fairground",
+        "--automatic-consent",
+    ]
+
+    process = subprocess.Popen(args=wallet_args)
+
+    return process
+
+
+class VegaServiceFairground(VegaService):
+    def __init__(
+        self,
+        warn_on_raw_data_access: bool = False,
```

Comment:
And then instead of the hardcoded strings we can have some `Config` object (thinking data class) with a default one for fairground/eventually main net which people can pass in.